projects/packages/my-jetpack/CHANGELOG.md

```diff
@@ -5,6 +5,27 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.6.8] - 2022-03-08
+### Added
+- Added connected plugins slugs to My Jetpack tracking events
+- Add link to jetpack.com in the footer
```

Comment:
I believe this is for My Jetpack as well. 

Could probably just do one general "My Jetpack: Various design related improvements"  or something and consolidate a few other entries as well.

Reply:
Looks like that is the my-jetpack changelog